docs/credits.md

```diff
@@ -0,0 +1,31 @@
+**NOTE!** This list is not complete. Please also see the git log itself for authors and committers and the Github statistics at:
+
+* https://github.com/rdiff-backup/rdiff-backup/people
+* https://github.com/rdiff-backup/rdiff-backup/graphs/contributors
+
+Project Lead / Maintainer History:
+
+* From August 2019 onwards the main driver for the project is Eric L. supported by Seravo.
```

Comment:
By Seravo and Minarca?

Reply:
I don't know much about Minarca, but naturally grateful for Patrik for his work. Regarding the "driver" part this refers to what happened last year when I and you resurrected this back. Let's update this again later when we see how things are in the long run.